starcheck/src/aca_load_review_cl.rst

```diff
@@ -328,11 +328,12 @@ Checks
 |ACA-040|Monitor commanding|X|X|Dither is disabled and enabled with correct|n/a |Failure to track|
 |       |                  | | |timing                                     |    |                |
 +-------+------------------+-+-+-------------------------------------------+----+----------------+
-|       |                  |X|X|                                           |    |AS: Possible    |
-|       |                  | | |                                           |    |Bright Star     |
-|ACA-041|Magnitude         | | |Slot MAXMAG (faint limit) - star MAG >= 0.3|n/a |Hold, GS:       |
-|       |                  | | |                                           |    |Reduced aspect  |
-|       |                  | | |                                           |    |quality         |
+|       |                  |X|X|                                           |    |AS: Reduced     |
+|       |                  | | |                                           |    |acq probability |
```

Comment:
Regarding the implications column, I think technically this would suggest that we should have starcheck set P_ACQ to 0 for acq stars with MAXMAG - MAG < 0.3 .  But I can PITEOG on this.